conf/messages.en

```diff
@@ -147,8 +147,10 @@ primaryContactDetails.tooManyAttempts.passcode.title = You have entered an incor
 primaryContactDetails.tooManyAttempts.passcode.1 = You have entered an incorrect passcode too many times
 primaryContactDetails.tooManyAttempts.passcode.2 = You must log out of you Government Gateway account and log back in to restart your registration.
 
-primaryContactDetails.phoneNumber.title = What is the primary contact’s telephone number?
-primaryContactDetails.phoneNumber.hint = This must be 11 numbers. For example 01234 567890.
+primaryContactDetails.phoneNumber.default = the primary contact
```

Comment:
I think we ought to have a single definition of the default "main contact".

Reply:
Removed